### ISSUE TITLE
hreflang issues at root pages

### ISSUE THREAD

**Issue by davydden** [lloc/Multisite-Language-Switcher#96]
as discussed [here](https://wordpress.org/support/topic/hreflang-issues-for-root-pages?replies=5#post-7823525), currently there is a problem with `hreflang` on root pages as read by google crawler.

As an example, for 

```
mydomain.com
mydomain.com/ru/
```

Google analytics complains 

> Google has detected that some pages on your site have implemented the rel-alternate-hreflang tag incorrectly. In particular, there seems to be a problem with incorrect language and region codes, or incorrect bi-directional linking (if page A links with hreflang to page B, there must be a link back from B to A as well).

with no return tags errors: 

```
'ru' - no return tags
'en' - no return tags
```

even though on both root pages we have

```
<link title="RU" href="http://mydomain.com/ru/" hreflang="ru" rel="alternate">
<link title="EN" href="http://mydomain.com/" hreflang="en" rel="alternate">
```

Note that these errors appear **only** on root pages; all other bi-lingual pages are read fine by google crawler.

The following, as proposed in the original discussion, might be a correct solution

```
<link href="http://mydomain.com/" hreflang="x-default" rel="alternate" />
<link title="EN" href="http://mydomain.com/" hreflang="en" rel="alternate">
<link title="RU" href="http://mydomain.com/ru/" hreflang="ru" rel="alternate">
```

**Comment by Vizzielli** [comment]
Same problem here. I've wordpress network setup, root redirect to it/ (and we have also en/ version).

Google said:
Language "en" - no return tags

It shows all it/ page under origin URL column and en/ version on alternative URL in the corresponding row.

All pages have:
<link rel="alternate" hreflang="en" href="http://www.comain.com/en/page-name/" title="en_GB" />
<link rel="alternate" hreflang="it" href="http://www.comain.com/it/translated-page-name/" title="it_IT" />

Would be possible that we must place the current hreflang in last position? (considering that it version work for me, what it we set en on the bottom in the english version?)

**Comment by lloc** [comment]
Please test ;)

**Comment by turboyz** [comment]
I got the same problem from Google Search Console

In this comment I **won't consider the blog root issue**, instead I'll focus on the problem related to single pages/posts that I saw @vizzielli talked about.

The blog I'm taking into account is ecobnb.com (with translation to .it .fr .de and also to es.ecobnb.com). 

I discovered that some of the few pages with the hreflang error indicated from Google are related to a missing cross linked article. 

**So this is not a code related issue but a content managing one.**

I try to be clearer w/ an example
I had an article in Italian. Translated to English, German and Spanish.

When some months later we translated to German, the content manager forgot (inside the german page) to link back to the Italian translation, instead, she linked back to the English and to the Spanish. 
Leaving a sort of "Gap" with the Italian page that created the Google error detection.

Once you add all the proper links, this issue disappears.
